src/main/java/ea/maps/America.java

```diff
@@ -4,15 +4,22 @@
 import ea.data.Resource;
 import org.springframework.stereotype.Component;
 
+import java.io.IOException;
 import java.util.*;
 
 @Component
 public class America extends BaseMap {
 
-  public static Map<Resource, List<Integer>> initializeResources() {
-    Map<Resource, List<Integer>> built = new HashMap<>();
+    final static String AMERICA_JSON_FILE = "src\\main\\resources\\america.json";
 
-    built.put(Resource.COAL, ImmutableList.of(
+    public void initializeCityList() throws IOException {
+        cities = MapFactory.getCityListFromJson(AMERICA_JSON_FILE);
```

Comment:
Use classpath.getResource to read the file as a string and pass the string into this method. This current method won't work within a jar.

Reply:
Should I still use the `AMERICA_JSON_FILE` path, instead of getResource, in the unit tests?